corehq/apps/users/models.py

```diff
@@ -1884,14 +1884,26 @@ def confirm_account(self, password):
 
     def get_case_sharing_groups(self):
         from corehq.apps.groups.models import Group
-        from corehq.apps.locations.models import get_case_sharing_groups_for_locations
+        from corehq.apps.locations.models import (
```

Comment:
Are there a specific reason why these imports are not at the top of the file? :)

Reply:
Some of them unfortunately can't be moved to prevent circular imports. But the rest I have gone ahead and moved up. Thanks for pointing that out. :+1: